lib/Parse/ParseDecl.cpp

```diff
@@ -896,43 +892,43 @@ bool Parser::parseDifferentiableAttributeArguments(
     return true;
   };
 
+  // Parse trailing comma, if it exists, and check for errors.
+  auto consumeIfTrailingComma = [&]() -> bool {
+    if (consumeIf(tok::comma)) {
```

Comment:
Negate the condition and early return.

Reply:
Thanks! Done in https://github.com/apple/swift/pull/21544/commits/adff0ab2125678960de1c27ba1d6eeede181b545